website/content/en/docs/faq.md

```diff
@@ -66,20 +45,20 @@ If you aren't sure what dependencies are required, start up a container using th
 
 ## I keep seeing errors like "Failed to watch", how do I fix this?
 
-If you run into the following error message:
+If you run into the following error message, it means that your operator is unable to watch the resoruce:
 
 ```
 E0320 15:42:17.676888       1 reflector.go:280] pkg/mod/k8s.io/client-go@v0.0.0-20191016111102-bec269661e48/tools/cache/reflector.go:96: Failed to watch *v1.ImageStreamTag: unknown (get imagestreamtags.image.openshift.io)
 {"level":"info","ts":1584718937.766342,"logger":"controller_memcached","msg":"ImageStreamTag resource not found.
 ```
 
-Then, it means that your Operator is unable to watch the resource. This scenario can be faced because the Operator does not have the permission [(RBAC)[rbac]] to `Watch` the resource, or may be the Schema from the API used, did not implement this verb. In this way the solution would be to grant the permission in the `role.yaml` , or when it is not  possible, use the [client.Reader][client.Reader] instead of the client provided.
+Using controller-runtime's split client means that read operations (gets and lists) are read from a cache, and write operators are written directly to the API server. To populate the cache for reads, controller-runtime initiates a `list` and then a `watch` even when your operator is only attempting to `get` a single resource. The above scenario occurs when the operator does not have an (RBAC)[rbac] permission to `watch` the resource. The solution is to grant permission in the `config/rbac/role.yaml` file.
```

Comment:
did you mean `write operations`? It currently reads as "and write operators are written directly to the API server"

Reply:
Yep! Good catch. I have waaaay too much muscle memory typing operator.

```suggestion
Using controller-runtime's split client means that read operations (gets and lists) are read from a cache, and write operations are written directly to the API server. To populate the cache for reads, controller-runtime initiates a `list` and then a `watch` even when your operator is only attempting to `get` a single resource. The above scenario occurs when the operator does not have an (RBAC)[rbac] permission to `watch` the resource. The solution is to grant permission in the `config/rbac/role.yaml` file.
```